editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5523,6 +5608,17 @@ CanvasItemEditor::CanvasItemEditor() {
 
 	main_menu_hbox->add_child(memnew(VSeparator));
 
+	local_space_button = memnew(Button);
+	local_space_button->set_theme_type_variation(SceneStringName(FlatButton));
+	main_menu_hbox->add_child(local_space_button);
+	local_space_button->set_toggle_mode(true);
+	local_space_button->set_pressed_no_signal(true);
+	local_space_button->connect(SceneStringName(toggled), callable_mp(this, &CanvasItemEditor::_button_toggle_local_space));
+	local_space_button->set_tooltip_text(TTRC("Use local space."));
```

Comment:
```suggestion
```
No reason to repeat the shortcut.

<img width="165" height="95" alt="Image" src="https://github.com/user-attachments/assets/afc8c285-57ac-4663-85bd-91fd7333c9c2" />